src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheck.java

```diff
@@ -67,13 +67,42 @@
  * &lt;module name="JavadocVariable"/&gt;
  * </pre>
  * <p>
+ * By default, this setting will report a violation if
+ * there is no javadoc for any scope member.
+ * </p>
+ * <pre>
+ * public class Test {
+ *   private int a; // violation, missing javadoc for private member
+ *
+ *   &#47;**
+ *   * Some description here
+ *   *&#47;
+ *   private int b; // OK
+ *   protected int c; // violation, missing javadoc for protected member
+ *   public int d; // violation, missing javadoc for public member
+ * }
+ * </pre>
+ * <p>
  * To configure the check for {@code public} scope:
  * </p>
  * <pre>
  * &lt;module name="JavadocVariable"&gt;
  *   &lt;property name="scope" value="public"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>This setting will report a violation if there is no javadoc for {@code public} member.</p>
+ * <pre>
+ * public class Test {
+ *   private int a; // OK
+ *
+ *   &#47;**
+ *   * Some description here
+ *   *&#47;
```

Comment:
Add one more space

Reply:
I have to add one line after "Some description here"?

Reply:
done